R/tableau.R

```diff
@@ -63,7 +64,6 @@ tableau_color_pal <- function(palette = "Tableau 10",
 #' and \funclink{scale_colour_gradient2_tableau} for the diverging continuous
 #' color scales from Tableu.
 #'
-#' @inheritParams ggplot2::scale_colour_hue
```

Comment:
Why did you drop this?

Reply:
It seems that all the parameters native to this function are already explicitly defined. The only remaining parameters being pulled from `scale_colour_hue` are the ones being passed to `discrete_scale` via the dots. 

I wasn't used to seeing that in other color palettes like `ggplot2::scale_color_brewer()` or `ggplot2::scale_color_viridis_d()`. But, I can see now that it's the intended format,  and taking it out makes it inconsistent with the rest of discrete color scales in `ggthemes`. 

I can put it back in, unless it's easier for you to discard that one deletion on your end. Let me know.

Thanks